vk_video_decoder/libs/VkVideoParser/VulkanVideoParser.cpp

```diff
@@ -1138,7 +1138,7 @@ int32_t VulkanVideoParser::BeginSequence(const VkParserSequenceInfo* pnvsi)
         detectedFormat.codec = pnvsi->eCodec;
         detectedFormat.frame_rate.numerator = NV_FRAME_RATE_NUM(pnvsi->frameRate);
         detectedFormat.frame_rate.denominator = NV_FRAME_RATE_DEN(pnvsi->frameRate);
-        detectedFormat.progressive_sequence = pnvsi->bProgSeq;
+        detectedFormat.progressive_sequence = pnvsi->bProgSeq & 0x1;
```

Comment:
I can't believe we are still using 32-bit value for bProgSeq.
Please revert the above change and change VkParserSequenceInfo::bProgSeq to
uint32_t progSeq : 1;
OR
bool progSeq;